src/olympia/devhub/utils.py

```diff
@@ -260,6 +261,12 @@ def __init__(self, file_, addon=None, listed=None, final_task=None):
             if waffle.switch_is_active('enable-yara'):
                 tasks_in_parallel.append(run_yara.s(file_.pk))
 
+            if waffle.switch_is_active('enable-customs'):
+                tasks_in_parallel.append(run_customs.s(file_.pk))
+
+            if waffle.switch_is_active('enable-wat'):
+                tasks_in_parallel.append(run_wat.s(file_.pk))
```

Comment:
I ran into some issues locally, could it be because we are modifying the Celery validation task with waffle? (i.e. it is not always exactly the same and maybe Celery and/or Django don't see the same validation task in the end)

Reply:
I hadn't thought of that. I don't think it should matter ? Ultimately the chain/chord differs yes, but the definition of the tasks included in them doesn't (the tasks are registered even if the waffle is off), so I'd say it should still work...